### ISSUE TITLE
x/pkgsite: "Frequently asked questions" hyperlinks have insufficient color contrast

### ISSUE THREAD

**Issue by julieqiu** [golang/go#40907]
The hyperlinks in `<h3>` of the site footer lack sufficient color contrast against the background of the `<div class="Questions-content">`. The hyperlink color should be made darker or the background of the div class should be made lighter.

/cc @jamalc

**Comment by Piyushhbhutoria** [comment]
I would like to work on it. Need some more insight for this issue

**Comment by jamalc** [comment]
Hi @Piyushhbhutoria thank you for your interest. This color contrast on the FAQ links was already fixed in a [previous commit](https://github.com/golang/pkgsite/commit/a1b9579989302b4f1ba71cd3bbb0ca0364ab09d4). I'll close the issue.
